app/src/models/account.ts

```diff
@@ -25,7 +25,10 @@ export class Account {
     /** The database id for this account */
     public readonly id: number,
     /** The friendly name associated with this account */
-    public readonly name: string
+    public readonly name: string,
+    public readonly plan?: {
```

Comment:
This can be extract as a type so we don't have to use the type annotation `{readonly name: Plan}` everywhere. I place an emoji next to the places I've spotted this type of usage.